src/utils/index.ts

```diff
@@ -1855,17 +1850,20 @@ export function getTokenAPRSyrup(syrup: SyrupInfo) {
     : 0;
 }
 
-export function getDQUICKAPYSyrup(syrup?: SyrupInfo) {
-  if (!syrup) return '0';
+export function useLairDQUICKAPY(lair?: LairInfo) {
+  const daysCurrentYear = getDaysCurrentYear();
+  const dQUICKPrice = useUSDCPriceToken(returnTokenFromKey('DQUICK'));
+  if (!lair) return '0';
   const dQUICKAPR =
-    (((Number(syrup.oneDayVol) * 0.04 * 0.01) /
-      Number(syrup.dQuickTotalSupply.toSignificant(6))) *
-      getDaysCurrentYear()) /
-    (Number(syrup.dQUICKtoQUICK.toSignificant(6)) * Number(syrup.quickPrice));
+    (((Number(lair.oneDayVol) *
+      GlobalConst.utils.DQUICKFEE *
+      GlobalConst.utils.DQUICKAPR_MULTIPLIER) /
+      Number(lair.dQuickTotalSupply.toSignificant(6))) *
```

Comment:
In the future we need to come up with a better way to determine what the function is being used for.  

For instance if we toSignificant 6 and this is being used as display that might make sense but if we were to use that when swapping it may not.

Reply:
I am going to remove all `toSignificant` through whole code